packages/grafana-ui/package.json

```diff
@@ -90,7 +90,7 @@
     "slate-plain-serializer": "0.7.10",
     "tinycolor2": "1.4.2",
     "tslib": "2.3.1",
-    "uplot": "1.6.19",
+    "uplot": "leeoniya/uPlot#4b8f468c998d984bf0921de265b4852f483bd046",
```

Comment:
Is this for testing? I'm not sure it's a good idea to publish Grafana NPM packages with dependencies that point to git commits. 🤔

Reply:
yes, it's temporary. i'm gonna cut an official uPlot release next week. (still have an unrelated bug to fix)

Reply:
Awesome! 👍